arangod/Agency/Store.h

```diff
@@ -173,7 +163,17 @@ class Store {
   /// and ignoring multiple subsequent forward slashes
   static std::vector<std::string> split(std::string const& str);
 
+#if defined(ENABLE_MOCKS)
+public:
```

Comment:
As far as I see this is not doing anything since we are anyway in `public` here.

Reply:
Furthermore, this means that in the mocks we would make everything below public, too. Do we want this?

Reply:
Good catch with the first part (it was already in the public section); flipping the condition now.